packages/web-components/fast-components/src/design-system-provider/design-system-provider.definition.ts

```diff
@@ -1,12 +1,521 @@
+import { DataType } from "@microsoft/fast-tooling";
 import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/esm/data-utilities/web-component";
+import { Direction } from "@microsoft/fast-web-utilities";
+import { fastDesignSystemDefaults } from "../fast-design-system";
 
 export const fastDesignSystemProviderDefinition: WebComponentDefinition = {
     version: 1,
     tags: [
         {
             name: "fast-design-system-provider",
             description: "The FAST design system provider element",
-            attributes: [],
+            attributes: [
+                {
+                    name: "use-defaults",
+                    description: "Use default values",
+                    type: DataType.boolean,
+                    default: true,
+                    required: false,
+                },
+                {
+                    name: "no-paint",
+                    description: "Do not paint background",
+                    type: DataType.boolean,
+                    default: undefined,
```

Comment:
This is the only attribute missing a default. Should this default to true like use-defaults?

Reply:
Whoops, you're right, although it should be false. Good catch.

```suggestion
                    default: false,
```